src/content-helper/editor-sidebar/title-suggestions/component.tsx

```diff
@@ -198,7 +198,7 @@ export const TitleSuggestionsPanel = (): React.JSX.Element => {
 
 		createNotice(
 			'error',
-			__( 'There was an error generating title suggestions.', 'wp-parsely' ),
+			error.message,
```

Comment:
Here we know that the error is a `ContentHelperError` object, so no special handling needed.